com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -67,23 +67,23 @@ For that reason, some parameters have no effect in path tracing, while others be
 
 ### Refraction model
```

Comment:
Hm. I re-wrote this section and it should have been merged. I'll go investigate this.